src/accessibility-statement.md

```diff
@@ -1,198 +1,83 @@
--->
+This accessibility statement applies to govuk-brand-guidelines.service.gov.uk.
 
-This website is run by \[name of organisation\]. We want as many people as possible to be able to use this website. For example, that means you should be able to:
+This website is run by the Government Digital Service (GDS). We want as many people as possible to be able to use this website. For example, that means you should be able to:
 
 - change colours, contrast levels and fonts using browser or device settings
 - zoom in up to 400% without the text spilling off the screen
 - navigate most of the website using a keyboard or speech recognition software
 - listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)
 
-We’ve also made the website text as simple as possible to understand.
-
-[AbilityNet](https://mcmw.abilitynet.org.uk/) has advice on making your device easier to use if you have a disability.
+AbilityNet has advice on [making your device easier to use](https://mcmw.abilitynet.org.uk/) if you have a disability.
 
 ## How accessible this website is
 
-<!--
-Note: use this section to provide a summary of accessibility issues that a disabled user can act on \- for example, avoid a particular section of the website, or request an alternative version rather than waste time trying to make it work with their assistive technology. Try to list in order of most impact to least impact.
--->
-
 We know some parts of this website are not fully accessible:
 
-- ...
+- Some videos don't have a description and don't say they have no audio.
+- Sets of colours are missing clear groupings.
 
 ## Feedback and contact information
 
-If you find any problems not listed on this page or think we’re not meeting accessibility requirements, contact: \[provide both details of how to report these issues to your organisation, and contact details for the unit or person responsible for dealing with these reports\].
-
-If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille:
+If you find any problems not listed on this page or think we’re not meeting accessibility requirements, email [govuk-brand-team@dsit.gov.uk](mailto:govuk-brand-team@dsit.gov.uk).
 
-- email \[email address\]
-- call \[phone number\]
-- \[add any other contact details\]
+If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille, email [govuk-brand-team@dsit.gov.uk](mailto:govuk-brand-team@dsit.gov.uk).
 
-We’ll consider your request and get back to you in \[number\] days.
-
-If you cannot view the map on our ‘contact us’ page, call or email us \[add link to contact details page\] for directions.
+We’ll consider your request and get back to you in \[TODO: number\] days.
```

Comment:
Waiting for response to @selfthinker's question to brand team.

Reply:
We've had the response now. I will remove the line entirely.